test/include/util/transaction_test_util.h

```diff
@@ -162,6 +216,24 @@ class LargeTransactionTestObject {
   void CheckReadsCorrect(std::vector<RandomWorkloadTransaction *> *commits);
 
  private:
+  /**
+   * Initializes a test object with the given configuration
```

Comment:
You need to break the documentation up into the builder as well. We don't document private things with doxygen